schema/appmodel/PDS.schema.xml

```diff
@@ -93,7 +93,9 @@
   <superclass name="SmartDaqApplication"/>
   <superclass name="ResourceSetDisableAND"/>
   <relationship name="configuration" class-type="DaphneConf" low-cc="one" high-cc="one" is-composite="no" is-exclusive="no" is-dependent="no"/>
-  <relationship name="detector_connections" description="List of detector components controlled by this application" class-type="FelixDetectorToDaqConnection" low-cc="zero" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
+  <relationship name="detector_connections" description="List of detector components controlled by this application" class-type="DetectorToDaqConnection" low-cc="zero" high-cc="many" is-composite="no" is-exclusive="no" is-dependent="no"/>
```

Comment:
the `low_cc` for both `detector_connections` and `hermes_modules_conf` is `zero`.
Does a `DaphneApplication` with those fields empty make sense?